playground/dockerfile/Dockerfile.AppHost/qots/qots.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+    "runtime"
```

Comment:
NIT: I think there are spaces here and not a tab and the go toolchain is going to yell during `go build` but maybe it will be cool about this?